collectors/COLLECTORS.md

```diff
@@ -501,6 +501,40 @@ the `go.d.plugin`.
 These collectors are developed and maintained by third parties and, unlike the other collectors, are not installed by
 default. To use a third-party collector, visit their GitHub/documentation page and follow their installation procedures.
 
+<details>
+<summary>Typical third party Python collector installation instructions</summary>
+<br>
+In general the below steps should be sufficient to use a third party collector.
+
+1. Download collector code file into folder expected by Netdata.
+2. Download default collector configuration file into folder expected by Netdata.
```

Comment:
Like previous, please, add a link for directories.

Reply:
For both you can use [this](https://learn.netdata.cloud/docs/agent/collectors/plugins.d#environment-variables) link.